inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/convolution/convolution_kernel_b_fs_yx_fsv_16_32_imad_dw.cpp

```diff
@@ -123,21 +111,137 @@ ConvolutionKernel_b_fs_yx_fsv_16_32_imad_dw::GetAutoTuneParams(const convolution
     if (index >= 0 && index < static_cast<int>(all_tune_params.size())) {
         return all_tune_params[index];
     }
-    AutoTuneParams tune_params;
-    tune_params.simd = 16;
-    if (params.output.GetLayout() == DataLayout::b_fs_yx_fsv16) {
-        tune_params.tile_x = std::min((size_t)16, params.output.X().v);
+
+    auto& output = params.output;
+
+    size_t fsv = 1;
+    if (output.GetLayout() == DataLayout::b_fs_yx_fsv16) {
+        fsv = 16;
     } else {
-        tune_params.tile_x = std::min((size_t)8, params.output.X().v);
+        fsv = 32;
```

Comment:
I think that you can change initialization to "size_t fsv = 32" and to remove "else" branch after that. Looks like you use only fsv = 16 and fsv = 32, so fsv = 1 don't really needed.

Reply:
Changed